src/tools/ci/docker/linux_centos8

```diff
@@ -0,0 +1,59 @@
+FROM centos:centos8
+# Fix the repos
+RUN \
+  sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-* && \
+  sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-* && \
+  yum -y install wget && \
+  wget 'http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-gpg-keys-8-3.el8.noarch.rpm' && \
+  rpm -i 'centos-gpg-keys-8-3.el8.noarch.rpm' && \
+  dnf -y --disablerepo '*' --enablerepo=extras swap centos-linux-repos centos-stream-repos && \
+  dnf -y distro-sync
```

Comment:
If we're using stream repos anyway, why don't we just start out with the stream container?

Reply:
Done.